a.py

```diff
@@ -0,0 +1,8 @@
+import torch
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch._dispatch.python import enable_python_dispatcher, patch_py_impls
+
+with enable_python_dispatcher(), patch_py_impls({
+    torch.ops.aten.matmul.default: {torch._C.DispatchKey.AutogradCPU: torch._C.DispatchKey.Autograd}
```

Comment:
this is the annoying thing where I can't just say Autograd because the C++ sided CompositeImplicitAutograd overrides me

Reply:
Hmmm why?
Do we define somewhere how alias keys work with python dispatch?

Reply:
No, it's not defined, it's an emergent property.

Essentially, take the dispatch key registrations from C++, and mash them up with the Python registrations (Python overwriting C++ if they had exactly the same dispatch key), and then compute the dispatch table from the mashup. So if you override Autograd from Python, that will only work if there wasn't a higher priority key overridden from C++.

Reply:
Ok makes sense!
The current behavior sounds ok then? The user should be well aware if the implementation they provide is meant to override the alias key or the specific key?
I guess what you want here is a KeySet to be able to override all the autograd keys at once?

Reply:
Yeah, so one way to hack this is introduce another Autograd alias key that has higher precedence than CompositeImplicitAutograd lol. It's just kind of a pain haha